requirements.txt

```diff
@@ -0,0 +1,28 @@
+attrs==23.2.0
+cffi==1.16.0
```

Comment:
What version of Python was used for generating this? Looks like cffi 1.16.0 dropped support for python 3.7, which is the stated version in the readme

Reply:
Python 3.11
Should I update readme to 3.11? Feels like that's the right thing to do, since 3.7 if end of life.

Reply:
Yes, please update the readme. I'll test this branch with python 3.11 to verify

Reply:
I've updated readme. Thank you for reviewing and testing.